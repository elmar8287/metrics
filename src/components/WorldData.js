import React from 'react';
import PropTypes from 'prop-types';

const WorldData = ({ totalConfirmed }) => (
  <section>
    <div>
      <div>
      
      </div>
      <aside>
        <h2>World</h2>
        <span>{`${totalConfirmed} cases`}</span>
      </aside>
    </div>
  </section>
);

WorldData.propTypes = {
  totalConfirmed: PropTypes.number,
};

WorldData.defaultProps = {
  totalConfirmed: 0,
};

export default WorldData;
