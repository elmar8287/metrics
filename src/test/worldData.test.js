import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configStore';
import WorldData from '../components/WorldData';

describe('Home page render correctly', () => {
  test('Home component renders correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <Provider store={store}>
            <WorldData />
          </Provider>
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
