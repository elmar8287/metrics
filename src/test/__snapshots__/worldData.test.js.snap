// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page render correctly Home component renders correctly 1`] = `
<section>
  <div
    className="main-info"
  >
    <aside
      className="world-info"
    >
      <h2>
        Worldwild Info
      </h2>
      <span>
        0 cases
      </span>
    </aside>
  </div>
</section>
`;
